src/libs/ValidationUtils.js

```diff
@@ -17,6 +18,21 @@ function isValidAddress(value) {
     return !CONST.REGEX.PO_BOX.test(value);
 }
 
+/**
+ * Used to validate a value that is "required".
+ *
+ * @param {*} value
+ * @returns {Boolean}
+ */
+function isValuePresent(value) {
+    if (_.isString(value)) {
+        return !_.isEmpty(value.trim());
+    }
+    if (_.isArray(value) || _.isObject(value)) {
+        return !_.isEmpty(value);
+    }
+    return Boolean(value);
+}
```

Comment:
NAB, I like this method. My one hesitation is whether the boolean case makes the name confusing b.c. a boolean with a `false` is technically "present" it's just `false`. 

For the boolean case we are just returning the boolean again not whether it has a value or not.

Reply:
I had the same concern about booleans, maybe it would be cleaner to consider `true` and `false` as present. Then, we just handle the case of the "terms and conditions" checkbox without this function.

Reply:
Ok, made a small change here. `false` and `true` (booleans) are considered "present". Therefore, we are not using this function to test the value for `hasNoConnectionToCannabis`.

Reply:
Ok cool I think this works for now. Probably we will want to think about this some more when we build a more generic form component and need a generic way to test that a field marked as "required" when a boolean will be tested for "existing" and "true".

Reply:
> Ok cool I think this works for now. Probably we will want to think about this some more when we build a more generic form component and need a generic way to test that a field marked as "required" when a boolean will be tested for "existing" and "true".

Yes, I think this is a good idea. Postpone thinking further about this until we have more cases or a more generic.